tests/shared/drive-constraints.spec.js

```diff
@@ -809,6 +975,19 @@ describe('Shared: DriveConstraints', function () {
       })
     })
 
+    describe('given the drive is pending', () => {
+      it('should return an empty list', function () {
```

Comment:
I think this `should return an empty list` should actually be `should return the pending error`, which would better match what the next unit-test does?